Terminal.Gui/Core/Application.cs

```diff
@@ -385,7 +411,7 @@ protected virtual void Dispose (bool disposing)
 
 		static void ProcessKeyEvent (KeyEvent ke)
 		{
-			if(RootKeyEvent?.Invoke(ke) ?? false) {
+			if (RootKeyEvent?.Invoke (ke) ?? false) {
```

Comment:
the space after invoke may be accidental?

Reply:
No it's how .editorconfig is configured.